src/io/parquet/write/mod.rs

```diff
@@ -56,6 +56,8 @@ pub struct WriteOptions {
     pub version: Version,
     /// The compression to apply to every page
     pub compression: CompressionOptions,
+    /// The size to flush a page, defaults to 1024 * 1024 if zero
+    pub data_pagesize_limit: Option<usize>,
```

Comment:
This should be `"The size to flush a page, defaults to 1024 * 1024 if None"`. Otherwise it could be interpreted as `Some(0)`.